### ISSUE TITLE
优化一些细节

### ISSUE THREAD

**Issue by tjleizeng** [linyiLYi/voice-assistant#16]
这个工程很棒，我优化了其中的一些细节。
1. 加入了回车跳过当前对话的功能；
2. 保存已经产生的对话到log.txt。

这是改动后的main.py文件，这个改动还需要pynput包以及在初次使用时在mac里按提示打开相应的权限。
[main.txt](https://github.com/linyiLYi/voice-assistant/files/13760984/main.txt)

**Comment by alfred-liang** [comment]
不知道为什么跑不起来34g的

**Comment by alfred-liang** [comment]
File "/Users/aliang/Downloads/voice-assistant-main/whisper/load_models.py", line 150, in convert
    return mx.array(model.detach().numpy())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
